src/test/java/org/folio/tlib/postgres/TenantPgPoolTest.java

```diff
@@ -216,4 +217,22 @@ public void testSSL(TestContext context) throws IOException {
     }));
   }
 
+  @Test
+  public void size() {
+    TenantPgPool pool = TenantPgPool.pool(vertx, "diku");
+    assertThat(pool.size(), is(0));
+  }
+
+  @Test
+  public void close(TestContext context) {
+    TenantPgPool pool = TenantPgPool.pool(vertx, "diku");
+    pool.close(context.asyncAssertSuccess());
+  }
+
+  @Test
+  public void connectHandler(TestContext context) {
+    TenantPgPool pool = TenantPgPool.pool(vertx, "diku");
+    pool.connectHandler(null);
+  }
+
```

Comment:
Can you add a comment how the tests are distributed over TenantPgPoolTest and TenantPgPoolImplTest?

Reply:
TenantPgPoolImplTest tests internals of TenantPgPoolImplTest, while  TenantPgPoolTest tests the  TenantPgPoolTest API